.github/workflows/build-test.yml

```diff
@@ -210,3 +214,33 @@ jobs:
                 CLOUD_PROVIDER: ${{ matrix.cloud }}
                 TARGET_DOCKER_TEST_IMAGE: ${{ matrix.image }}
               run: ./ci/test.sh
+
+    display_code_coverage:
+        needs: test-mac-required
+        name: Display code coverage
+        runs-on: macos-latest
+        strategy:
+          matrix:
+            cloud: [ 'AWS' ]
+        steps:
+          - uses: actions/checkout@v1
+          - uses: actions/setup-node@v1
+            with:
+              node-version: '12.x'
```

Comment:
maybe let's use node LTS - it's just to show the results

Reply:
I working here. I'm not sure that is good way change version only this one place and have different

Reply:
ok